app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
We generally don't keep unused resources since they increase App Size without a reason. Just keep the ones you are using for now 👌